.code-samples.meilisearch.yaml

```diff
@@ -447,3 +479,30 @@ geosearch_guide_sort_usage_2: |-
   client.index('restaurants').search('', {
     sort: ['_geoPoint(48.8583701,2.2922926):asc', 'rating:desc'],
   })
+security_guide_search_key_1: |-
+  const client = new MeiliSearch({ host: "http://localhost:7700", "apiKey": apiKey })
```

Comment:
Again about the quotes haha, what do you think about making all of them using the same quotes, be `'` or `"` but for all the samples on the js side 🌗